### DISCUSSION TITLE
Images attached to a collection disappears after Refreshing, even It succefully uploaded to Minio.

### DISCUSSION THREAD

**Discussion by RaulGrosmam** [outline/outline#4790] [Self Hosting]
Hello,

I'm using a local Kubernetes to host my Outline Instance (Minikube).
Everything works fine, except for uploading an Image to any collection on Outline.
I can upload files and everything else, except images directly to the Collection.

All Images uploads normally to Minio and keeps appearing normally to the collection.
But If the page is refreshed, all the images disappears from the collection, but It keeps uploaded to Minio.

Image uploaded
![image](https://user-images.githubusercontent.com/49730433/215164944-145f87cd-75ab-4006-98cd-722bfd6b1073.png)

All HTTP requests shows 200 when the image is uploaded.
![image](https://user-images.githubusercontent.com/49730433/215163380-ff870556-8e90-4b80-80cc-cb1fa9b6e54b.png)

The image uploaded on Minio
![image](https://user-images.githubusercontent.com/49730433/215163634-fff4116f-a703-4353-8e2a-b4ae42aa0b03.png)

Page after refreshing
![image](https://user-images.githubusercontent.com/49730433/215165011-a6c9fe65-70bd-4676-9c4d-ba656a8fdf24.png)

All Requests after refreshing the page
![image](https://user-images.githubusercontent.com/49730433/215164019-755f9b2e-2d82-4219-b8f7-2209f48be581.png)

There's no message error, the images just disappears from the collection.

Here's all my config

Configmap: (I'm not using secret)
![image](https://user-images.githubusercontent.com/49730433/215164299-9620cd91-2ca8-494c-8551-4bbe1dddc0ae.png)

Deployment:
![image](https://user-images.githubusercontent.com/49730433/215164480-0793ae76-7eff-415b-8063-5c40d824b7f6.png)

Outline Pod Log:
![image](https://user-images.githubusercontent.com/49730433/215164601-5ae1db86-aa61-4d48-8809-a02cac5342a3.png)

Minio Pod Log:
![image](https://user-images.githubusercontent.com/49730433/215164692-8f428d07-32c0-4915-b033-8969b9d1fcda.png)

**Comment by tommoor** [comment]
Based on your screenshots it doesn't look like it uploaded successfully, the image is in the faded placeholder state which it shows while uploading. And the Minio screenshot is of a folder not an uploaded file.

**Comment by RaulGrosmam** [comment]
Here's the image uploaded to Minio.
![image](https://user-images.githubusercontent.com/49730433/215459107-4e1f389a-d5d0-4661-acd8-d6ce7104e402.png)

Also, every image stay faded on collections.
There's a way to troubleshoot it?

**Reply by tommoor** [reply]
Is your domain name a UUID? It looks like it… that could be causing the issue.

**Reply by RaulGrosmam** [reply]
Well, I'm using everything local with Minikube. Except for Bucket_URL. I'm using ngrok to expose minio's k8s service, since I don't have any Ingress controller on my Cluster.
Also on Config Map I'm using localhost as URL. (Already tested using an exposed URL aswell)
You think that might be the problem?

**Comment by IgnisDa** [comment]
@tommoor I can get minio to work only if I set bucket policy to public. Is this the only way?

![image](https://user-images.githubusercontent.com/60938164/216243818-4d59cb0b-4b21-4b91-80ab-0bfa097ee137.png)
